### ISSUE TITLE
Scala 2.13 and dependency updates

### ISSUE THREAD

**Issue by ruippeixotog** [wookietreiber/scala-chart#34]
Hi! Can you update this project to support Scala 2.13, updating its downstream dependencies if needed? Thank you!

**Comment by wookietreiber** [comment]
I don't really have the capacity any more to care about scala-chart. I have just transferred scala-chart to its own organization and invited you @ruippeixotog to it. I'm doing so in the hopes that if there is enough interest in the project, it can continue this way as a community project. 

I have also invited you @Sciss because I have seen you have kept scala-chart alive in your fork. 

Let me know if you need anything done with the organization.

**Comment by Sciss** [comment]
Yes, I published an artifact for 2.13: https://search.maven.org/artifact/de.sciss/scala-chart_2.13

**Comment by Sciss** [comment]
Ah, I see. Should I merge back then the changes? I changed package base to `de.sciss.chart` to avoid confusion.

**Comment by wookietreiber** [comment]
I guess. I used **com.github.wookietreiber** as organization before while publishing. Maybe this would also have to be changed to reflect that the project is going to be community maintained.

**Comment by ruippeixotog** [comment]
Sounds good to me - I probably won't have the time to build a lot of new features myself but I'm fine with keeping dependencies up-to-date and reviewing incoming PRs.

Do you want to push your changes to https://github.com/scala-chart/scala-chart, @Sciss? I can set up the project in Sonatype and we can start publishing it under `com.github.scalachart` if you like.

**Comment by Sciss** [comment]
@wookietreiber I am happy to continue working on this main repository. You need to make a decision whether I should merge my version, which uses base package `de.sciss.chart`, or if we should change to another package name. Since we don't have hold on TLD scala-chart.org or similar, I suggest to simply leave this package which would also correspond to the published artifact's group-id.

**Comment by wookietreiber** [comment]
@Sciss Good to hear! 

As far as I remember, back in the day sonatype was fine with me using **com.github.wookietreiber** as group-id without having a distinct TLD for the project, as long as they could confirm that my account and the repository existed on GitHub. I also think, that they don't check if the package names in the JAR matches the group-id, consistency does not seem to be required, although it may be conventional.

I guess, we could create (at least for now) a minimal GitHub Page for the **scala-chart** GitHub organization and use that as the group-id. I think the address would be **https://scala-chart.github.io** and the corresponding group-id would be **io.github.scala-chart**. 

Back in the day, I chose **scalax.chart** as package name because, at that time, the **scalax** prefix was used by a few other extension library projects. I don't have an overview of how that evolved. I think, we could also change the package name according to the new group-id for consistency. 

Does that sound reasonable? I mean, I don't really want to decide anything on my own, while I'm turning this project over to the community ;)

**Note:** What I don't know is how publishing is handled in a case like this. How is a new group-id/project registered and who may publish to it?

**Comment by Sciss** [comment]
@wookietreiber AFAIK, you need to open a new ticket - https://central.sonatype.org/pages/ossrh-guide.html - for a new group-id. They will usually confirm within a few days. Probably you can already add me as a second author authorised to publish.

I would avoid having github in the package or group-ids, though. For example, [Scala STM](http://scala-stm.org/) uses `org.scala-stm`, but then Nathan seems to have paid for scala-stm.org which just redirects to his [github page](https://nbronson.github.io/scala-stm/). Don't know how difficult it is to register a tld?
